demisto_sdk/commands/content_graph/strict_objects/generic_type.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import Field
+
+from demisto_sdk.commands.content_graph.strict_objects.base_strict_model import (
+    StrictGenericIncidentType,
+)
+
+
+class StrictGenericType(StrictGenericIncidentType):  # type:ignore[valid-type,misc]
```

Comment:
instead of `type:ignore`ing here, we can use a model that has these two fields, and `create_model` to inherit from both it and StrictGenericIncidentType